<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DPadEditDialog</class>
 <widget class="QDialog" name="DPadEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Presets:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="presetsComboBox">
         <property name="minimumSize">
          <size>
           <width>282</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mouse (Normal)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mouse (Inverted Horizontal)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mouse (Inverted Vertical)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mouse (Inverted Horizontal + Vertical)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Arrows</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Keys: W | A | S | D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NumPad</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="joyModeComboBox">
         <property name="minimumSize">
          <size>
           <width>282</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Standard</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Eight Way</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>452</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Configure Mouse Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="margin">
         <number>8</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QCheckBox" name="changeMouseSpeedsCheckBox">
            <property name="text">
             <string>Change Speeds</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="changeTogetherCheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Change Together</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Horizontal Speed:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="horizontalSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>300</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item alignment="Qt::AlignRight">
             <widget class="QLabel" name="horizontalSpeedLabel">
              <property name="text">
               <string>1 = 20 pps</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Vertical Speed:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="verticalSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>300</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item alignment="Qt::AlignRight">
             <widget class="QLabel" name="verticalSpeedLabel">
              <property name="text">
               <string>1 = 20 pps</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DPadEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DPadEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
